.clj-kondo/config.edn

```diff
@@ -642,8 +641,7 @@
    metabase.test/with-gtaps                                                     macros.metabase-enterprise.sandbox.test-util/with-gtaps
    metabase.test/with-persistence-enabled                                       macros.metabase.test.persistence/with-persistence-enabled
    metabase.test/with-temp-env-var-value                                        macros.metabase.test.util/with-temp-env-var-value
-   metabase.test/with-temporary-raw-setting-values                              macros.metabase.test.util/with-temporary-raw-setting-values
-   toucan.models/defmodel                                                       macros.toucan.models/defmodel}}
```

Comment:
Since we imported the Toucan 2 Kondo config in https://github.com/metabase/metabase/pull/27742 we don't actually need our own version anymore.